File: src/main/java/de/flavormate/ba_entities/openFoodFacts/model/OpenFoodFactsProduct.java
/* Licensed under AGPLv3 2024 */
package de.flavormate.ba_entities.openFoodFacts.model;

/**
 * Represents a product retrieved from the OpenFoodFacts API.
 *
 * @param nutriments The nutritional information of the product.
 */
public record OpenFoodFactsProduct(OpenFoodFactsNutriments nutriments) {}

File: src/main/kotlin/de/flavormate/extensions/openFoodFacts/api/models/OpenFoodFactsProduct.kt
/* Licensed under AGPLv3 2024 - 2026 */
package de.flavormate.extensions.openFoodFacts.api.models

/**
 * Represents a product retrieved from the OpenFoodFacts API.
 *
 * @param nutriments The nutritional information of the product.
 */
data class OpenFoodFactsProduct(val nutriments: OpenFoodFactsNutriments)
